### PR TITLE
Mbp 137 - Changes to support multiple envs

### PR DESCRIPTION
**Pull request by modusemerson**
Mbp 137 - Changes to support multiple envs

Here is the gutlab job demonstrating that the tests work
https://gitlab.product.mbox-dev.io/iac/dev/-/jobs/19045

### CODE REVIEW

File: ui-tests/tests/src/config.ts [hunk 1]
```diff
@@ -11,18 +11,26 @@
...
-dotenv.config();
+//dotenv.config();
 
 function ensureEnv(e: string): string {
   const result = process.env[e];
   assert.notStrictEqual(typeof result, 'undefined', `Required ${e} to be set in the environment`);
   return result as string;
 }
 
+let config1;
```
Comment (gibaros) [inline ui-tests/tests/src/config.ts:22]: Silly comment, but can we use a variable that provides more context than 'config1'? Maybe swapConfig or tmpConfig or something else.

File: ui-tests/tests/src/config.ts [hunk 2]
```diff
@@ -41,5 +49,39 @@ export const config = {
       password: 'test',
     },
   },
-  voodooTimeoutMs: 30000,
+  voodooTimeoutMs: 30000
 };
+
+}else if(process.env.ENV === "other"){
```
Comment (gibaros) [inline ui-tests/tests/src/config.ts:55]: Just like we have an if above for "local" should we have an explicit if condition for PRODUCTION...just want to make sure that there is no confusion when we are dealing a a PRODUCTION environment.

**Review by gibaros — APPROVED** [review]
Please review the 2 comments added below. Otherwise lgtm 👍

File: ui-tests/tests/src/config.ts (hunk 2 above)
Comment (modusemerson) [reply ui-tests/tests/src/config.ts:55]: Hi @gibaros  these tests are not intented to be run in production env so I think we dont need to cover this point

File: ui-tests/tests/src/config.ts (hunk 1 above)
Comment (modusemerson) [reply ui-tests/tests/src/config.ts]: I just made few changes for this pls revie again thanks @gibaros

**Review by gibaros — APPROVED** [review]
lgtm

**Pull request merged**